src/pages/IstioConfigDetails/IstioConfigDetailsPage.tsx

```diff
@@ -37,6 +37,8 @@ class IstioConfigDetailsPage extends React.Component<RouteComponentProps<IstioCo
     };
     activeFilters.push(namespaceFilter);
     if (addObjectTypeFilter) {
+      console.log('TODELETE ' + this.props.match.params.objectType);
+      console.log('TODELETE ' + dicIstioType[this.props.match.params.objectType]);
```

Comment:
We should remove the console.log. What do you think?

Reply:
Ouch, yes, thanks @aljesusg; indeed, I put the TODELETE to not forget :-)